ddl/util/util.go

```diff
@@ -62,17 +62,21 @@ func LoadDeleteRanges(ctx context.Context, safePoint uint64) (ranges []DelRangeT
 		if row == nil {
 			break
 		}
-		startKey, err := hex.DecodeString(row.Data[2].GetString())
+		sVal, _ := row.GetString(2)
```

Comment:
Does GetString need to return error? Looks like it's no necessary.

Reply:
It's a bool value for `isNull`.
It follows the pattern of `expression.EvalXXX`.
Because there is another dedicated method `IsNull`, so the second return value can be removed.
I planned to remove it in the future PR.